docs/dataset/object-detection.md

```diff
@@ -0,0 +1,366 @@
+    and click `Import` to upload the model results.
+
+    <figure markdown>
+    ![Model Results Upload](../assets/images/task-od-upload-results-light.gif#only-light)
+    ![Model Results Upload](../assets/images/task-od-upload-results-dark.gif#only-dark)
+    <figcaption markdown>Model Results Upload</figcaption>
+    </figure>
+
+    It's also important to note down the evaluation configuration of the model being uploaded, for example:
+    `iou_threshold: 0.5` or `iou_threshold: 0.3`.
+
+    <div class="grid" markdown>
+    <div markdown>
+    <figure markdown>
+        ![Example of a Model Configuration](../assets/images/task-od-model-config-dark.png#only-dark)
+        ![Example of a Model Configuration](../assets/images/task-od-model-config-light.png#only-light)
+        <figcaption>Example of a Model Configuration</figcaption>
+    </figure>
+    </div>
+    <figure markdown>
+    ![Setting Evaluation Configurations](../assets/images/task-od-evaluation-config-dark.gif#only-dark)
+    ![Setting Evaluation Configurations](../assets/images/task-od-evaluation-config-light.gif#only-light)
+    <figcaption>Setting Evaluation Configurations</figcaption>
+    </figure>
+    </div>
+
+    You can repeat the above steps with all the other model files availible.
+
+=== "SDK"
+
+    The example code contains a script [`object_detection_2d/upload_results.py`](https://github.com/kolenaIO/kolena/blob/trunk/examples/dataset/object_detection_2d/object_detection_2d/upload_results.py)
+    which will process raw model results from `s3://kolena-public-examples/coco-2014-val/transportation/results/raw/`
+    and upload model results using the `upload_object_detection_results` function.
+
+    ```shell
+    poetry run python3 object_detection_2d/upload_results.py yolo_x
+    poetry run python3 object_detection_2d/upload_results.py faster_rcnn
+    ```
+
+    Results for two models named `yolo_x` and `faster_rcnn` will appear after the upload is complete.
+
+### Step 3: Define Object Detection Quality Standards
+
+Once your dataset and model results are uploaded, comparing models across different scenarios and metrics becomes
+very easy. Simply set up your [Quality Standards](../dataset/core-concepts/quality-standard.md) by
+[defining test cases](../dataset/quickstart.md#define-test-cases) and
+[defining performance metrics](../dataset/quickstart.md#define-metrics).
+
+<figure markdown>
+  ![Setting up object detection Quality Standards](../assets/images/task-od-qs-light.gif#only-light)
+  ![Setting up object detection Quality Standards](../assets/images/task-od-qs-dark.gif#only-dark)
+  <figcaption markdown>Setting up object detection Quality Standards</figcaption>
+</figure>
+
+Rather than uploading raw inferences with metrics yourself, if you upload the
+[TP / FP / FN](../metrics/tp-fp-fn-tn.md) results from your own [bounding box matcher](../metrics/geometry-matching.md)
```

Comment:
Aside from using their own bounding box matcher, we can also mention our util functions that can compute the necessary model results columns from raw inference. 

- [compute_object_detection_results](https://docs.kolena.com/reference/pre-built/object-detection-2d/?h=upload_#kolena._experimental.object_detection.compute_object_detection_results) 
- [upload_object_detection_results](https://docs.kolena.com/reference/pre-built/object-detection-2d/?h=upload_#kolena._experimental.object_detection.upload_object_detection_results)

Reply:
I still think we should surface these two options (1. use your own bbox matcher and give us the TP / FP / FN results and 2. upload your raw inferences and use Kolena's bbox matcher) very clearly under the step #3 section.